logconfig/collector.go

```diff
@@ -38,6 +38,9 @@ func NewCollector(options *LogOptions) *Collector {
 
 // Archive creates ZIP archive containing all node log files.
 func (c *Collector) Archive() (outputFilepath string, err error) {
+	if c.options.Filepath == "" {
+		return "", errors.New("file logging is disabled, can't retrieve logs")
+	}
```

Comment:
This method is still possible insecure as it may archive not only logs files. I suggest to filter file names to ensure that they start with `.mysterium-log*`.

Reply:
I believe this comment points at the wrong line. This fragment specifically checks for disabled file logging.

Sure, I'll check what we can do.

Reply:
It should be `mysterium-node.log`, not `mysterium-log`.

`logFilepaths` actually checks for this. The problem was that the filename was not being set (so there was nothing to check against). Everything should work now as originally intended.